CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.2.6+2
```

Comment:
[optional] IMO "support secret tokens in the 2.3.x Dart SDK" could be considered a new feature instead of a bug fix.

Reply:
It's a bug fix in the sense that we are smashing over the paths. It just so happened to work <2.3 because the observatory URLs never contained a path.